### PR TITLE
[TGBOT-79] Process command get customerid

### PR DESCRIPTION
**Pull request by Sav4ykVladzislau**
https://schneidervs.atlassian.net/browse/TGBOT-79

### CODE REVIEW

**Review by schneidervs — APPROVED** [review]

**Review by yurasava — DISMISSED** [review]

File: stoservice/src/main/java/com/tutrit/stoservice/utils/GetMap.java [hunk 1]
```diff
@@ -41,18 +32,11 @@ private static String getObject(String inputMsg) {
...
-                temp[0] = temp[0].replace("\"", "");
-                temp[1] = temp[1].replace("\"", "");
-                map.put(temp[0], temp[1]);
-            }
+    private static String getKey(String inputMsg) {
+        if (inputMsg.contains("-") && inputMsg.contains("-d")) {
+            return inputMsg.substring(inputMsg.indexOf('-') + 1, inputMsg.indexOf("-d")).trim();
         }
-        return map;
+        return "null";
```
Comment (AlexanderYarokhovich) [inline stoservice/src/main/java/com/tutrit/stoservice/utils/GetMap.java:39]: I would not return null as a string

File: stoservice/src/main/java/com/tutrit/stoservice/utils/GetId.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.tutrit.stoservice.utils;
+
+public class GetId {
+    private  String getId(String inputMsg) {
+        if (inputMsg.contains("-") && inputMsg.contains("-id")) {
+           return inputMsg.substring(inputMsg.indexOf('-') + 1, inputMsg.indexOf("-id")).trim();
```
Comment (AlexanderYarokhovich) [inline stoservice/src/main/java/com/tutrit/stoservice/utils/GetId.java:6]: use .strip

**Review by AlexanderYarokhovich — APPROVED** [review]

File: stoservice/src/main/java/com/tutrit/stoservice/utils/GetId.java (hunk 2 above)
Comment (schneidervs) [reply stoservice/src/main/java/com/tutrit/stoservice/utils/GetId.java:6]: Use GetIdFromMap

File: stoservice/src/main/java/com/tutrit/stoservice/utils/GetMap.java (hunk 1 above)
Comment (schneidervs) [reply stoservice/src/main/java/com/tutrit/stoservice/utils/GetMap.java:39]: I fixed it in my branch

**Review by schneidervs — DISMISSED** [review]

**Review by schneidervs — DISMISSED** [review]

**Review by schneidervs — APPROVED** [review]

**Comment by AlexanderYarokhovich** [comment]
Good job

**Comment by schneidervs** [comment]
> 

Alex, approving reviews

**Review by yurasava — DISMISSED** [review]

**Review by AlexanderYarokhovich — DISMISSED** [review]

**Review by yurasava — APPROVED** [review]

**Review by AlexanderYarokhovich — APPROVED** [review]

**Review by schneidervs — APPROVED** [review]

**Pull request merged**